### PR TITLE
Some grammar and stylistic suggestions

### PR DESCRIPTION
**Pull request by gkoz**
These are the changes making the post easier to parse for me

### CODE REVIEW

File: _posts/2015-09-21-Contributor-s-story.md [hunk 1]
```diff
@@ -6,28 +6,28 @@ categories: [front, org]
...
+As the time passed, gkoz became my second on `Gtk-rs` (congrats to him \o/). He started a big refactoring of all the code structure. We decided then to create a website to let the users of `Gtk-rs` follow changes (and breaking changes) more easily. For some reasons detailed in a previous post, the organization's name was changed from `rust-gnome` to `Gtk-rs`.
 
-To resume all this post, I must say that I never thought I would take the leadership of this project and that I would get the help of such amazing people (contributors and users!). It was for me a way to learn a new part of `Rust` and I hope I'll be able to contribute to it a lot in the future!
+To conclude this post, I must say that I never thought I would take the leadership of this project and that I would get the help of such amazing people (contributors and users!). For me it was a way to learn a new part of `Rust` and I hope I'll be able to contribute to it a lot in the future!
```
Comment (GuillaumeGomez) [inline _posts/2015-09-21-Contributor-s-story.md]: "and I hope I'll be able to contribute to it a lot in the future" -> "and I hope I'll be able to continue contributing to it a lot in the future"

Comment (gkoz) [inline _posts/2015-09-21-Contributor-s-story.md]: Isn't this a mouthful though? "I hope to contribute a lot in the future"?

Comment (GuillaumeGomez) [inline _posts/2015-09-21-Contributor-s-story.md]: It's more like "I'm not contributing right now but I hope I will in the future". It seems not appropriate.

Comment (gkoz) [inline _posts/2015-09-21-Contributor-s-story.md]: Anyway the PR doesn't touch this sentence so you could tweak it afterwards.

**Pull request merged**